### ISSUE TITLE
Missing tooltips

### ISSUE THREAD

**Issue by nucleus-ffm** [nucleus-ffm/foss_warn#115]
**Describe the bug**
Any button without text should have a tooltip to allow the application to be used with a screen reader.

**To Reproduce**
Tooltips are missing for: 
- the sort button 
- the button to add a new place
- ~maybe the card to open a place~
- ~maybe the card to open the detail view of a warning~

**Expected behavior**
Each icon button should have a tooltip
